usaspending_api/broker/management/commands/update_duns.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime
+from django.core.management.base import BaseCommand
+from django.db import connections, transaction
+from django.db.models import Max
+
+from usaspending_api.recipient.models import DUNS
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
+from usaspending_api.etl.management.load_base import load_data_into_model
+
+import logging
+
+logger = logging.getLogger('console')
+
+
+class Command(BaseCommand):
+
+    def gather_new_duns(self, db_cursor, update_date, latest_broker_duns_id):
+        new_duns_query = "SELECT * FROM duns " \
+                         "WHERE updated_at > \'" + str(update_date) + "\' AND " \
+                         "duns_id > " + str(latest_broker_duns_id)
+        db_cursor.execute(new_duns_query)
+        new_duns = dictfetchall(db_cursor)
+
+        update_duns_query = "SELECT * FROM duns " \
+                            "WHERE updated_at > \'" + str(update_date) + "\' AND " \
+                            "duns_id <= " + str(latest_broker_duns_id)
+        db_cursor.execute(update_duns_query)
+        update_duns = dictfetchall(db_cursor)
+
+        return new_duns, update_duns
+
+    def add_duns(self, new_duns, update_date):
+        new_records = []
```

Comment:
Please add log statements for the record counts to be added/updated in these methods